KNOWN_ISSUES.md

```diff
@@ -0,0 +1,40 @@
+Known Issues in Coq
+===================
+
+> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.
+
```

Comment:
I believe that quote is attributable to de Saint-Exupery, not the Coq dev team. :-)